pkg/cli/query.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2023 Chainguard, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cli
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"text/template"
+
+	"chainguard.dev/melange/pkg/build"
+	"github.com/spf13/cobra"
+)
+
+func Query() *cobra.Command {
```

Comment:
I like this!

I've been using [`yq`](https://github.com/mikefarah/yq) a bit for this kind of thing, which doesn't support Go templates, but it does have a pretty extensive query language, like `jq`, and can even update things in-place (if you accept its formatting decisions).

Reply:
Credit to @kaniini on the `query` idea.